### DISCUSSION TITLE
How to add your own fill pattern?

### DISCUSSION THREAD

**Discussion by khameleonium** [tannerhelland/PhotoDemon#545] [Q&A] [answered]
![image](https://github.com/tannerhelland/PhotoDemon/assets/94788083/8df222be-6ec1-4726-a3e3-640623f4c681)

**Comment by tannerhelland** [comment, marked as answer]
Hello @khameleonium .  I'm afraid those patterns are supplied by Windows itself, and there is not currently a way to provide your own pattern in PhotoDemon.

Other users have requested this feature in https://github.com/tannerhelland/PhotoDemon/issues/546 .  I will look at adding it to the program soon.

Thank you for asking about this!

**Reply by khameleonium** [reply]
Thank you for your reply and for your work! This is one of the best graphic editors out there! Of course, it's not really suitable for complex professional tasks, but for everyday tasks of a normal user it's just PERFECT!!!! I personally use it for quick editing of video game sprites, transcoding multiple images into one format, quick removal of backgrounds, painting, etc. And it's just SUPER! The editor loads in a second, is easy to transport, requires no installation and is very functional. I bow down to your talent!
I was especially surprised that the source code is written in VB6.
